### ISSUE TITLE
DX11 Realtime Lighting

### ISSUE THREAD

**Issue by tarkus1971** [TheGameCreators/GameGuruRepo#194]
Will there be any work done on the dynamic lights now that DX11 is in.  More lights, better shadows, spot, flood and more editable flashlight options.

**Comment by LeeBamberTGC** [comment]
Now that we are on a DirectX 11 foundation and Shader Model 5.0 we can do a lot more :)  Just a question of what and when, and thanks to the collaborative power of GitHub I think we can move on some of these ideas soon.

**Comment by tarkus1971** [comment]
Agreed, GitHub was a great idea Lee, really bringing stuff together in a much more cohesive and productive way. Awesome.

**Comment by UnknowNomad** [comment]
Speaking of which, realtime global illumination system will help those murky GameGuru games.
Video about difference in visual quality can be seen here:
[https://www.youtube.com/watch?v=FUYUNgr22m8](url)
